### PR TITLE
Fixes a lotta spelling and grammatical errors, also changes the EZ Brig cell door access to all-access meaning brigged crewmembers can access these cells

### PR DESCRIPTION
**Pull request by pumpkintheproot**
## About the Pull Request

This PR aims to repair the spelling and grammatical errors in the buttons on the 4th level of the map. It also changes the EZ Brig cells to be all-access, meaning brigged people will be able to enter these cells, as guards hardly interact with the people in here. There is also flashes in all the cells to stop prisoners hiding in the cells to avoid being flashed. This PR also adds bolts to the cell doors to make sure any pesky prisoners stay in a cell.
## Why It's Good For The Game
Guards hardly interact with prisoners after brigging them, placing them in, and after the time is done, getting them out. The cells were placed as all-access because, considering the probability of one actually being brigged, and the danger of said criminal, it's rare that a guard will actually have to use these cells to lock prisoners somewhere safe. Even though there's no place to keep them, there is no dangerous items outside of this ~~and if guards choose they can just straight jacket or bring them to CDCZ if they are really being a BITCH!!!~~ theres now door bolts so you can trap them inside

Being a brigged crewmember is boring, especially in EZ when you have a small room with nothing in it but a few tables. 
## Changelog

:cl:
qol: EZ brig cells now allow prisoners to enter, deleting ID access and allowing prisoners who presumably have their ID removed to enter
fix: grammatical errors, mostly including "outter" to "outer" and a button with the label "Checkkpoint" to "Checkpoint"
change: flashes to cells, meaning with the new ID changes prisoners cannot hide in cells to avoid being flashed
change: bolt switches to cells, meaning you can close or open the cells bolting system from the hit of a button!

/:cl:

### CODE REVIEW

**Comment by pumpkintheproot** [comment]
### **Do NOT merge this!!**
I'm unable to get this working locally so i'll have to investigate the cause.

**Comment by pumpkintheproot** [comment]
### Working on local test server
This is ready to be merged, everything should work as always unless anyone has any objections to the changes

File: maps/site53/site53-4.dmm [hunk 1]
```diff
@@ -2438,8 +2435,7 @@
 	d2 = 2;
 	icon_state = "1-2"
 	},
-/turf/simulated/floor/plating,
-/area/site53/maintenance/upper_surface)
```
Comment (aidan-bowman) [inline maps/site53/site53-4.dmm]: wtf why did you delete upper surface maintenance

File: maps/site53/site53-4.dmm [hunk 2]
```diff
@@ -4236,7 +4227,7 @@
 /obj/machinery/button/blast_door{
 	dir = 4;
 	id_tag = "EZ Lobby Outter Lockdown";
```
Comment (aidan-bowman) [inline maps/site53/site53-4.dmm]: change the id_tag too

**Review by cheesePizza2 — CHANGES_REQUESTED** [review]
your changelog sucks, rewrite it

File: maps/site53/site53-4.dmm (hunk 1 above)
Comment (pumpkintheproot) [reply maps/site53/site53-4.dmm:2442]: mustve been on accident while using strongdmm, sorry bout that

**Comment by pumpkintheproot** [comment]
> your changelog sucks, rewrite it

i have never wrote changelogs but i changed the changelog to be more detailed 😺

**Comment by pumpkintheproot** [comment]
#1482 fixes issue

**Review by pumpkintheproot — COMMENTED** [review]
okay idk what is going on but this is goobering hard

**Comment by pumpkintheproot** [comment]
AAA HOLY FUCVK FINALLY

**Comment by pumpkintheproot** [comment]
moving to new pr
